cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -69,6 +70,8 @@ renderShelleyTxCmdError err =
       "Error while writing signed shelley tx: " <> renderApiError apiError
     ShelleyTxWriteUnsignedTxError apiError ->
       "Error while writing unsigned shelley tx: " <> renderApiError apiError
+    ShelleyTxSubmitError txSubmitRes ->
+      "Error while submitting Shelley tx: " <> textShow txSubmitRes
```

Comment:
We should write a `renderTxSubmitError` function here to provide more context with the errors vs just showing them.